package.json

```diff
@@ -214,6 +214,7 @@
     "express": "4.16.2",
     "extract-text-webpack-plugin": "3.0.2",
     "fastclick": "1.0.6",
+    "filesize": "3.6.1",
```

Comment:
when you update the package.json file, this should update the yarn.lock file. Can you make sure that's added too?